pages/workflows/[id].js

```diff
@@ -0,0 +1,133 @@
+// node_modules
+import Link from "next/link";
+import PropTypes from "prop-types";
+// components
+import Attribution from "../../components/attribution";
+import Breadcrumbs from "../../components/breadcrumbs";
+import {
+  DataArea,
+  DataItemLabel,
+  DataItemValue,
+  DataItemValueUrl,
+  DataPanel,
+} from "../../components/data-area";
+import { EditableItem } from "../../components/edit";
+import JsonDisplay from "../../components/json-display";
+import ObjectPageHeader from "../../components/object-page-header";
+import PagePreamble from "../../components/page-preamble";
+import SeparatedList from "../../components/separated-list";
+// lib
+import buildBreadcrumbs from "../../lib/breadcrumbs";
+import errorObjectToProps from "../../lib/errors";
+import FetchRequest from "../../lib/fetch-request";
+import AliasList from "../../components/alias-list";
+import buildAttribution from "../../lib/attribution";
+import { isJsonFormat } from "../../lib/query-utils";
+
+export default function Workflow({ workflow, attribution = null, isJson }) {
+  return (
+    <>
+      <Breadcrumbs />
+      <EditableItem item={workflow}>
+        <PagePreamble />
+        <ObjectPageHeader item={workflow} isJsonFormat={isJson} />
+        <JsonDisplay item={workflow} isJsonFormat={isJson}>
+          <DataPanel>
+            <DataArea>
+              <DataItemLabel>Name</DataItemLabel>
+              <DataItemValue>{workflow.name}</DataItemValue>
+              <DataItemLabel>Description</DataItemLabel>
+              <DataItemValue>{workflow.description}</DataItemValue>
```

Comment:
Just noticed `description` isn’t mandatory, so this can display the “Description” label with no value. You have to put this label and value behind a conditional.